packages/generator-adp/src/translations/generator-adp.i18n.json

```
@@ -59,6 +65,7 @@
         "appParameterMissing": "Could not create an extension project because the application parameters are missing. Please select a different application.",
         "destinationInfoMissing": "Could not create an extension project because the destination info is missing. Please select a different system.",
         "systemNotFound": "System not found or not supported: {{system}}. Please check the system exists.",
-        "applicationNotFound": "Application not found or not supported: {{appName}}. Please check the application exists."
+        "applicationNotFound": "Application not found or not supported: {{appName}}. Please check the application exists.",
+        "callingDeployGen": "Failed to invoke the Fiori deployment sub-generator."
```

```suggestion
        "callingDeployGen": "Failed to invoke the SAP Fiori deployment sub-generator."
```
Is there something the user can do? Check the logs? Try again? Contact support?

---

This was removed totally